tests/test_crontab.py

```diff
@@ -169,46 +169,54 @@ def test_remove_job_but_keep_anything_else():
 @patch('tests.cron.cron_job')
 def test_run_no_arg_format1_job(method_to_call):
     crontab = Crontab()
-    crontab.run_job('4f30993ab69a8c5763ce55f762ef0433')
+    assert crontab.run_job('4f30993ab69a8c5763ce55f762ef0433')
     method_to_call.assert_called_with()
 
 
 @override_settings(CRONJOBS=[('*/1 * * * *', 'tests.cron.cron_job', '> /home/myhome/logs/cron_job.log')])
 @patch('tests.cron.cron_job')
 def test_run_no_arg_format1_job_with_suffix(method_to_call):
     crontab = Crontab()
-    crontab.run_job('53e6fe5b66bd870e396d574ba1503c33')
+    assert crontab.run_job('53e6fe5b66bd870e396d574ba1503c33')
     method_to_call.assert_called_with()
 
 
 @override_settings(CRONJOBS=[('*/1 * * * *', 'tests.cron.cron_job', [1, 'two'])])
 @patch('tests.cron.cron_job')
 def test_run_args_only_format2_job(method_to_call):
     crontab = Crontab()
-    crontab.run_job('369f8418b0f8cf1fff78c547516aa8e0')
+    assert crontab.run_job('369f8418b0f8cf1fff78c547516aa8e0')
     method_to_call.assert_called_with(1, 'two')
 
 
 @override_settings(CRONJOBS=[('*/1 * * * *', 'tests.cron.cron_job', [1, 'two'], {'test': 34, 'a': 's2'})])
 @patch('tests.cron.cron_job')
 def test_run_format2_job(method_to_call):
     crontab = Crontab()
-    crontab.run_job('13e8169dffe273b8b0c5f8abe1b6f643')
+    assert crontab.run_job('13e8169dffe273b8b0c5f8abe1b6f643')
     method_to_call.assert_called_with(1, 'two', test=34, a='s2')
 
 
 @override_settings(CRONJOBS=[('*/1 * * * *', 'tests.cron.cron_job', [1, 'two'], dict(), 'some suffix')])
 @patch('tests.cron.cron_job')
-def test_run_args_only_format2_job(method_to_call):
+def test_run_args_only_format3_job(method_to_call):
```

Comment:
I got warnings about this being a duplicate name with the above test, so I renamed it